src/features/Courses/CoursesTable/columns.jsx

```diff
@@ -30,6 +37,45 @@ const columns = [
     Header: 'Students invited',
     accessor: 'numberOfPendingStudents',
   },
+  {
+    Header: '',
+    accessor: 'className',
+    cellClassName: 'dropdownColumn',
+    Cell: ({ row }) => {
```

Comment:
This header shouldn't have this arrows, I think with a little css could be hidden ( not sure if this package has an api do this ) 

![e](https://github.com/Pearson-Advance/frontend-app-institution-portal/assets/52179095/2f5fa75b-bfdf-4d6f-be17-79d6c86bffc7)